packages/core/src/plugins/mermaid.ts

```diff
@@ -0,0 +1,38 @@
+import cheerio from 'cheerio';
+import { PluginContext, FrontMatter } from './Plugin';
+
+const DEFAULT_CDN_ADDRESS = 'https://unpkg.com/mermaid@10/dist/mermaid.esm.min.mjs';
```

Comment:
I was just thinking why are we using the cdn version of mermaid. I read yongliang's old PR and he did that too?

Is the reasoning be such that we want Vue to have access of mermaid and it is not trivial to do it? Either way, we should document the rationale!

Reply:
> I was just thinking why are we using the cdn version of mermaid. I read yongliang's old PR and he did that too?

Question: Why not? :)

Reply:
My original rationale was that it would be easier to manage mermaid version update, and there shouldn't be loading speed concern using a cdn

Reply:
I seee. Thanks @tlylt for clarification. On top of my head, I was just concerned about reliability of cdn. I think it would cache by the browser so my concerns were trivial